docs/core/deploying/ready-to-run.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Ready to Run
+description: Learn what ReadyToRun is and why you should consider using it as part of the deployment.
+author: davidwr
+ms.author: davidwr
+ms.date: 09/21/2020
+---
+# ReadyToRun Compilation
+Startup time and application latency can be improved by compiling your application assemblies as ReadyToRun (R2R) format. R2R is a form of ahead-of-time (AOT) compilation.
+
+R2R binaries improve startup performance by reducing the amount of work the just-in-time (JIT) compiler needs to do as your application loads. The binaries contain similar native code compared to what the JIT would produce. However, R2R binaries are larger because they contain both intermediate language (IL) code, which is still needed for some scenarios, and the native version of the same code. R2R is only available when you publish an app that targets specific runtime environments (RID) such as Linux x64 or Windows x64.
+
+To compile your project as ReadyToRun, do the following:
+
+01. Add the `<PublishReadyToRun>` setting to your project:
+
+    ```xml
+    <PropertyGroup>
+      <PublishReadyToRun>true</PublishReadyToRun>
+    </PropertyGroup>
+    ```
+
+02. Publish an application. For example, this command creates an app for the 64-bit version of Windows:
+
+    ```dotnetcli
+    dotnet publish -c Release -r win-x64
+
+    ```
+
+## Impact of using the ReadyToRun feature
+
+### General impact
+Ahead of time compilation has somewhat complex and unpredictable performance impacts on application performance. In general the size of an assembly will grow to between 2 to 3 times larger. This increase in the physical size of the file may reduce the performance of loading the assembly from disk, and increase working set of the process. However, in return the number of methods compiled is typically reduced substantially. The result is that most applications that consist of a substantial amount of code receive large performance benefits from enabling ReadyToRun. Applications which have small amounts of code will likely not experience a significant improvement from enabling ReadyToRun, as the .NET framework assemblies have already been precompiled with ReadyToRun.
```

Comment:
'Unpredictable' sounds too strong here.  I would try rewording, e.g.,
> Ahead-of-time compilation has somewhat complex performance impact on application performance, which may be difficult to predict.

Also, could we please use consistent spelling of 'ahead-of-time' (regarding hyphens) at all places?